client/my-sites/site-settings/disconnect-site/why/disconnect-reasons-suggestion.jsx

```diff
@@ -0,0 +1,172 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import SelectDropdown from 'components/select-dropdown';
+import { isJetpackSite } from 'state/sites/selectors';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+
+class DisconnectReasonsSuggestion extends Component {
+	constructor( props ) {
+		super( props );
+		const initialState = {
+			childSelected: 'default',
+			compactButtons: false,
+		};
+		this.state = initialState;
+	}
+	componentWillReceiveProps() {
+		const state = {
+			childSelected: 'default',
+		};
+		this.state = state;
+	}
+
+	_logReasonFollowUp = ( option ) => {
+		this.setState( {
+			childSelected: option.value,
+		} );
+	}
+	setOptions = ( optionValue ) => {
```

Comment:
A question on this method name: are we really setting something? Shouldn't this be `getOptions` or something similar instead?